src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java

```diff
@@ -190,6 +199,31 @@
  *         the very same way as it is done by HTTPS. If "false" the host name is not verified at all. Per default the
  *         hostname is verified (true).
  *     </li>
+ *
+ *     <li>
+ *         <b>ch.hevs.cloudio.endpoint.token</b><br>
+ *
+ *         This property contains a single-use authentication token for the endpoint. This token is used to retrieve
+ *         certificates using the cloud.iO Rest API.
+ *
+ *     </li>
+ *     <li>
+ *         <b>ch.hevs.cloudio.endpoint.tokenPropertiesPath</b><br>
+ *
+ *         This property is used to specify a path where the retrieved certificates and .properties file from token
```

Comment:
This property is used to specify a path where the retrieved certificates and .properties file from the provisioning process will be saved.<br>

This property is optional and taken into account only if token based provisioning is used.
The default value is <b>~/.config/cloud.io/</b>.

IMHO `~/.config/cloud.io/` is a better default as we are not sure the the endpoint is running as root.